File: c/p2529.c
int maximumCount(int* nums, int numsSize) {
    int negatives = 0;
    int positives = 0;

    for (int i = 0; i < numsSize; i++) {
        if (nums[i] > 0) {
            positives += 1;
        } else {
            if (nums[i] < 0) {
                negatives += 1;
            }
        }
    }

    if (negatives > positives) {
        return negatives;
    }
    return positives;
}

File: src/p2529.rs
pub fn run() {
    for i in [

    ] {
        println!("{}", maximum_count(i));
    }
}

pub fn maximum_count(nums: Vec<i32>) -> i32 {
    let mut p = 0;
    let mut n = 0;

    for i in nums {
        if i > 0 {
            p += 1;
        } else if i < 0 {
            n += 1;
        }
    }

    p.max(n)
}